schemas/includes/unknown_entity_type.json

```diff
@@ -52,6 +52,12 @@
     "mailing_address": {
       "$ref": "address.json"
     },
+    "identifiers": {
+      "type": ["array", "null"],
```

Comment:
Ok good point re. existing bots. Let's just leave it in then. We can resolve the inconsistency globally in a future version of the schema.